src/richie/apps/core/templates/richie/base.html

```diff
@@ -1,10 +1,43 @@
 {% load i18n cms_tags menu_tags static sekizai_tags %}
 <!doctype html>
-<html>
+<html lang="{{ LANGUAGE_CODE }}">
     <head>
+        {% spaceless %}
         <title>{% block title %}{{ SITE.name }}{% endblock title %}</title>
         <meta name="viewport" content="width=device-width,initial-scale=1">
-        {% block meta %}{% endblock meta %}
+        {% block meta %}
+            {% block meta_index_rules %}
+                <meta name="robots" content="index,follow,all">
+            {% endblock meta_index_rules %}
+
+            {% block meta_html %}
+                {% page_attribute "meta_description" as meta_description %}
+                <meta name="description" content="{% block meta-description %}{% if meta_description and meta_description != "None" %}{{ meta_description }}{% endif %}{% endblock %}">
+                <meta name="keywords" content="{% block meta-keywords %}{% endblock %}">
```

Comment:
Same question. Maybe remove this meta if we have nothing to set for the moment?

Reply:
Removed